tests/test_inference_async_client.py

```diff
@@ -59,199 +44,6 @@ def tgi_client() -> AsyncInferenceClient:
     return AsyncInferenceClient(model="openai-community/gpt2")
 
 
-@pytest.mark.vcr
-@pytest.mark.asyncio
-@with_production_testing
-async def test_async_generate_no_details(tgi_client: AsyncInferenceClient) -> None:
-    response = await tgi_client.text_generation("test", details=False, max_new_tokens=1)
-    assert isinstance(response, str)
-    assert response == "."
```

Comment:
```suggestion
@pytest.mark.skip(reason="Skipping VCR tests until better solution is implemented.")
async def test_async_generate_no_details(tgi_client: AsyncInferenceClient) -> None:
    response = await tgi_client.text_generation("test", details=False, max_new_tokens=1)
    assert isinstance(response, str)
    assert response == "."
```

Could we keep all the tests but skip them? This way if/when we add proper tests (e.g. semi-live tests with a cached proxy) we should be able to quickly get back on track.